unit/testing-utils/catch.hpp

```diff
@@ -7326,8 +7326,9 @@ namespace Catch {
             catch( std::exception& ex ) {
                 return ex.what();
             }
-            catch( std::string& msg ) {
-                return msg;
+            catch(const std::string &msg)
+            {
+              return msg;
```

Comment:
I'm not sure this is to be touched - AFAIK the header has been copied as-is from somewhere else?